docs/components/dropdown.md

```diff
@@ -0,0 +1,271 @@
+# Dropdown
+
+**KDropdown** is a dropdown component used to show actions or options in popout. This only uses CSS and is for simple use cases. For more dynamic use cases check out [KPop](/components/popover.html) 
+
+<KDropdown
+  toggle-text="Items"
+  :items="['one', 'two', 'three']"/>
+
+```vue
+<KDropdown
+  toggle-text="Items"
+  :items="['one', 'two', 'three']"/>
+```
+
+## Props
+
+### items - required
+An array of items to show in the dropdown. Can be passed as strings or objects.
+
+<Komponent :data="{ items: [{name: 'item1'}, {name: 'item2'}, {name:'item3'}] }" v-slot="{ data }">
+  <KDropdown
+    toggle-text="Items"
+    :items="data.items" />
+</Komponent>
+
+```vue
+<template>
+  <KDropdown
+    toggle-text="Items"
+    :items="items"/>
+  </template>
+<script>
+export default {
+  data () {
+    return {
+      items: [
+        {name: 'item1'},
+        {name: 'item2'},
+        {name:'item3'}
+      ]
+    }
+  }
+}
+</script>
+```
```

Comment:
I wish there was an automatic way to catch when these don't match, or infer them

Reply:
good catch this file should have been removed